src/BenchmarkDotNet.Diagnostics.Windows/EtwProfiler.cs

```diff
@@ -102,32 +101,44 @@ private void Start(DiagnoserActionParameters parameters)
 
             try
             {
+                if (parameters.Process?.StartInfo.FileName != null)
+                {
+                    foreach (var provider in config.Providers)
+                    {
+                        provider.options.ProcessNameFilter = new List<string>() { parameters.Process?.StartInfo.FileName };
+                    }
+                }
+
                 userSession = new UserSession(parameters, config, CreationTime).EnableProviders();
+                heapSession = new HeapSession(parameters, config, CreationTime).EnableProviders();
```

Comment:
important: with this change `HeapSession` is always enabled, even if using `EtwProfiler` as standalone. Could you please extend the `EtwProfilerConfig` with a new flag and make this optional and used by default only be the `NativeMemoryDiagnoser` ?

Reply:
TODO